### PR TITLE
Issue templates

### PR DESCRIPTION
**Pull request by tvbeek**

### CODE REVIEW

**Comment by kylekatarnls** [comment]
@tvbeek Is it still Work In Progress or ready?

**Comment by tvbeek** [comment]
> 
> 
> @tvbeek Is it still Work In Progress or ready?

Yes, I miss the documentation one. I did start it but got interrupted. I will update here if done.

**Comment by tvbeek** [comment]
I have added the last issue template. So I'm ready with it. Please provide any feedback.

File: .github/4_Documentation_and_Website_Issue.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: "Documentation & Website Issue"
+about: 'Do you see something incorrect or do you miss some information in the documentation or the website. Tell us.'
+---
+
+## Description
+Describe what you miss or is incorrect on the website our documentation, be as explicit as possible.  
+If possible add an url to the incorrect website documentation.
```
Comment (kylekatarnls) [inline .github/4_Documentation_and_Website_Issue.md]: a URL

File: .github/4_Documentation_and_Website_Issue.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+## Checks before submitting
+* [ ] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/phpmd/phpmd/issues)
+* [ ] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/phpmd/phpmd/pulls)
+* [ ] Tell if you have the possibility to create a Pull Request to solve this issue.
+* [ ] If possible add an url to the incorrect website or documentation.
```
Comment (kylekatarnls) [inline .github/4_Documentation_and_Website_Issue.md]: add a URL

File: .github/1_Bug.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+- PHPMD version: #.#.#
+- PHP Version: #.#.#
+- Installation type: (phar file/ composer)
+
+### Expected Behavior
+Explain the result that you expected.
+
+### Current Behavior
```
Comment (ravage84) [inline .github/1_Bug.md]: I would move that up.

In my mind, the order goes:

1. What has happened (current hbehavior)
2. What did I expect to happen (expected behavior)
3. Steps to reproduce )help us to help you)

File: .github/1_Bug.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+### Steps To Reproduce:
+Explain **all** the steps you did to create this bug so we can reproduce it.
+
+## Checks before submitting
+* [ ] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/phpmd/phpmd/issues)
+* [ ] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/phpmd/phpmd/pulls)
+* [ ] I have added every step to reproduce the bug.
+* [ ] If possible I added relevant code examples.
+* [ ] This issue is about 1 bug and nothing more.
```
Comment (ravage84) [inline .github/1_Bug.md]: May be we should add "Have a descriptive title" with a good example?

File: .github/2_Feature_Proposal.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+name: "Feature Proposal"
+about: 'Propose a new feature'
+
+---
+
+## Description
+Describe the proposal, explain what it does and why.  
+Try to describe the benefits but also the cons of the proposal.
```
Comment (ravage84) [inline .github/2_Feature_Proposal.md]: cons -> consequences. Not everything is a negative/contra argumement. Also, I wouldn't use abbreviated language. Not everybody knows what "cons" are.

File: .github/2_Feature_Proposal.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+## Checks before submitting
+* [ ] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/phpmd/phpmd/issues)
+* [ ] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/phpmd/phpmd/pulls)
+* [ ] Tell if you have the option to provide the code for this proposal.
+* [ ] This issue is about 1 feature proposal and nothing more.
```
Comment (ravage84) [inline .github/2_Feature_Proposal.md]: May be we should add "Have a descriptive title" with a good example?

File: .github/3_Question.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
+name: "Question"
+about: 'This repository is for bugs and feature proposals. If you have any question please visit or community on Gitter to ask the questions: https://gitter.im/phpmd/community'
+---
+
+This repository is for bugs and feature proposals.
```
Comment (ravage84) [inline .github/3_Question.md]: +,", only."

File: .github/3_Question.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+
+If you have any question or need support please visit our Gitter channel: https://gitter.im/phpmd/community
```
Comment (ravage84) [inline .github/3_Question.md]: Once we have paid support, we can add a link to a list of paid support providers here, too.

File: .github/4_Documentation_and_Website_Issue.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
+---
+name: "Documentation & Website Issue"
+about: 'Do you see something incorrect or do you miss some information in the documentation or the website. Tell us.'
+---
+
+## Description
+Describe what you miss or is incorrect on the website or our documentation. Be as explicit as possible.
```
Comment (ravage84) [inline .github/4_Documentation_and_Website_Issue.md]: We should add a link to the website & the documentation here.

File: .github/3_Question.md (hunk 8 above)
Comment (tvbeek) [reply .github/3_Question.md]: True, but currently we don't have that :)

File: .github/1_Bug.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
+---
+name: "Bug"
+about: 'Report a problem with PHPMD'
+---
+
+- PHPMD version: #.#.#
+- PHP Version: #.#.#
+- Installation type: (phar file/ composer)
```
[the remark this answers is not in the document]
Comment (tvbeek) [reply .github/1_Bug.md]: Good point, I have added it to the template with some examples.

**Comment by ravage84** [comment]
I'm going to merge this once 2.7.0 is out.

**Comment by tvbeek** [comment]
> 
> 
> .github directory should be added to .gitattributes as export-ignore so it will not be downloaded by composer.

Good detail, I totally forgot it.

**Pull request merged**